tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -543,10 +543,11 @@ def docomplicated(suptitle=None):
 # Each item has a layoutbox associated with it. The nesting of gridspecs
 # created with `.GridSpecFromSubplotSpec` can be arbitrarily deep.
 #
-# Each `.Axes` has *two* layoutboxes.  The first one ``ax._layoutbox``
-# represents the outside of the Axes and all its decorations (i.e. ticklabels,
-# axis labels, etc.).  The second layoutbox corresponds to the Axes'
-# `ax.position`, which sets where in the figure the spines are placed.
+# Each ``~matplotlib.axes.Axes` has *two* layoutboxes.  The first one,
```

Comment:
Again, why did you change these?  The original worked just fine in the main docs...  https://matplotlib.org/tutorials/intermediate/constrainedlayout_guide.html#sphx-glr-tutorials-intermediate-constrainedlayout-guide-py